.gitignore

```diff
@@ -6,3 +6,4 @@ target/
 **/secret.conf
 **/.DS_Store
 .vscode/
+.scalafmt.conf
```

Comment:
IMO we should add a config, actually...

the one we had in https://github.com/pitgull/pitgull/blob/fb09bada100d18f41753a6de4363788053e84b71/.scalafmt.conf seems good. Feel free to format in a follow-up PR